client/actions/post.js

```diff
@@ -0,0 +1,69 @@
+// Action Types
+const LOADPOST = 'LOADPOST';
+const LOADPOSTS = 'LOADPOSTS';
+const FAILEDTOLOADPOST = 'FAILEDTOLOADPOST';
+const REMOVEPOST = 'REMOVEPOST';
+const FAILEDTODELETEPOST = 'FAILEDTODELETEPOST';
+
+// Actions
+export const _loadPost = (post) => ({
+    type: LOADPOST,
+    post,
+});
+
+export const _loadPosts = (posts) => ({
+    type: LOADPOSTS,
+    posts
+})
+
+const _failedtoDeleteUser = () => ({ type: FAILEDTODELETEPOST });
+
+const _removePost = () => ({ type: REMOVEPOST });
+
+const _failedToLoadPost = () => ({ type: FAILEDTOLOADPOST });
+
+// Thunks
+export const loadPost = (postId) => async (dispatch, _, axios) => {
+    try {
+        const { data: post } = await axios.get(`/api/posts/${postId}`);
+        if (post) {
+            dispatch(_loadPost(post));
+        } else {
+            dispatch(_failedToLoadPost());
+        }
+    } catch (err) {
+        dispatch(_failedToLoadPost());
+        console.error(err);
+    }
+};
+
+export const removePost = (postId) => async (dispatch, _, axios) => {
+    try {
+        const { data: deleted } = await axios.delete(`/api/post/deletePost/${postId}`);
```

Comment:
I don't see this api endpoint in /apiRoutes/post? Is it elsewhere?

Reply:
I think @CB6670 is just setting up the reducer here in this PR and the route will be created on the server in another PR.  I wouldn't worry to much about this just yet.

@CB6670: I see that you started creating routes on the server for Post. Maybe just add the deletepost route with a note "To be completed" just so that it exists...

Reply:
Ok, sounds good and yeah Justin I actually created a seperate issue for myself for the delete route since I forgot it in my first post PR.

Reply:
Gotcha!